lib/widgets/action_sheet.dart

```diff
@@ -562,7 +562,7 @@ void showMessageActionSheet({required BuildContext context, required Message mes
   // So we rely on the fact that isComposeBoxOffered for any given message list
   // will be constant through the page's life.
   final messageListPage = MessageListPage.ancestorOf(pageContext);
-  final isComposeBoxOffered = messageListPage.composeBoxController != null;
+  final isComposeBoxOffered = messageListPage.composeBoxState?.controller != null;
```

Comment:
Could this just be

```suggestion
  final isComposeBoxOffered = messageListPage.composeBoxState != null;
```

As the dartdoc of `composeBoxState` states:

```dart
  /// The [ComposeBoxState] for this [MessageListPage]'s compose box,
  /// if this [MessageListPage] offers a compose box and it has mounted,
  /// else null.
  ComposeBoxState? get composeBoxState;
```

Checking `composeBoxState?.controller != null` seems a bit confusing use of the API to me.

Reply:
Oh, indeed.